WebServer/microservices/db/dbresolver.py

```diff
@@ -0,0 +1,83 @@
+from psycopg2 import connect, errors
+from aiohttp import web
+import asyncio
+import json
+
+# Sends a query to the database and returns the response.
+# Inspired by: #https://kb.objectrocket.com/postgresql/python-and-postgresql-docker-container-part-2-1063
+def executeQuery(query,*inputs):
+    # declare connection instance
+    conn = connect(
+        dbname = "traffic_annonymisation_db",
+        user = "postgres",
+        host = "postgres",
+        password = "postgres",
+        port = 5432
+    )
+
+    results = []
+    error = None
+
+    try:
+        # declare a cursor object from the connection
+        cursor = conn.cursor()
+
+        # execute an SQL statement using the psycopg2 cursor object.
+        # By seperating the query and the inputs, psycopg2 does sanitation.
+        cursor.execute(query,inputs)
+        conn.commit()
+
+        # enumerate() over the PostgreSQL records
+        for i, record in enumerate(cursor):
+            results.append(record)
+
+    except errors.InvalidTextRepresentation as e:
+        error = e
+
+    except errors.UniqueViolation as e:
+        error = e.diag.message_detail 
+
+    except errors.InvalidDatetimeFormat as e:
+        error = "The data time format is invalid. An example of a correct date is: '2020-04-30 11:06:50'."
+    
+    
+    finally:
+        # close the cursor object to avoid memory leaks
+        cursor.close()
+
+        # close the connection as well
+        conn.close()
```

Comment:
Er disse kommentarer ikke meget unødvendige, da koden siger hvad der sker, bortset fra  "By seperating the query..."

Reply:
![image](https://user-images.githubusercontent.com/23058931/78566935-36173a80-7820-11ea-8a89-779188470e1f.png)